molecule/molecule-testing/src/main/kotlin/app/cash/molecule/testing/moleculeTesting.kt

```diff
@@ -52,6 +52,11 @@ fun <T> testMolecule(
 ) = testMolecule(body, timeoutMs.milliseconds, validate)
 
 @ExperimentalCoroutinesApi
+@Deprecated(
+  """Use moleculeFlow or launchMolecule with an 'Immediate' clock and then use Turbine to test.
+
+If you believe this is insufficient please let us know at https://github.com/cashapp/molecule/issues/97.""",
```

Comment:
No `trimMargin()` in annotation literals :(

Reply:
weak